reference/glossary.md

```diff
@@ -0,0 +1,445 @@
+- [Scrcpy](#scrcpy)
+- [Session](#session)
+- [Snap](#snap)
+- [Software Rasterization](#swrast)
+- [Stream agent](#stream-agent)
+- [Stream gateway](#stream-gateway)
+- [Streaming stack](#streaming-stack)
+- [Swrast](#swrast)
+- [Ubuntu Advantage for Applications](#ubuntu-advantage-for-applications)
+- [Ubuntu One](#ubuntu-one)
+- [Watchdog](#watchdog)
+- [Web dashboard](#web-dashboard)
+- [WebRTC](#webrtc)
+
+[/Details]
+
+## Definitions
+
+<a name="addon"></a>
+### Addon
+
+A piece of code that can be used to extend and customise images in Anbox Cloud.
+
+See https://discourse.ubuntu.com/t/addons/25293.
+
+<a name="aws"></a>
+### Amazon Web Services (AWS)
+
+A cloud platform provided by Amazon that can be used to host Anbox Cloud.
+
+See [the AWS website](https://aws.amazon.com/).
+
+<a name="ams-node-controller"></a>
+### AMS Node Controller
+
+A service that runs on every LXD node and puts the appropriate firewall rules in place when a container is started or stopped.
+
+<a name="ams-sdk"></a>
+### AMS SDK
+
+An SDK that provides Go code for connecting to AMS through the exposed REST API.
+
+See [AMS SDK](https://discourse.ubuntu.com/t/anbox-cloud-sdks/17844#ams-sdk).
+
+<a name="anbox"></a>
+### Anbox
+
+An open-source project that provides a container-based approach to boot a full Android system on a regular GNU/Linux system. Anbox Cloud uses Anbox.
+
+See [the Anbox project on GitHub](https://github.com/anbox/anbox).
+
+<a name="aam"></a>
+### Anbox Application Manager (AAM)
+
+A utility (`aam`) that is installed in the Android image and that can be used to back up and restore application data.
+
+See https://discourse.ubuntu.com/t/back-up-and-restore-application-data/24183.
+
+<a name="aar"></a>
+### Anbox Application Registry (AAR)
+
+A central repository for applications created in Anbox Cloud. Using an AAR is very useful for larger deployments to keep applications in sync.
+
+See https://discourse.ubuntu.com/t/anbox-application-registry-aar/17761.
+
+<a name="anbox-cloud"></a>
+### Anbox Cloud
+
+A rich software stack that enables you to run Android applications in the cloud for all kinds of different use cases, including high-performance streaming of graphics to desktop and mobile client devices.
+
+See https://discourse.ubuntu.com/t/about-anbox-cloud/17802.
+
+<a name="anbox-cloud-appliance"></a>
+### Anbox Cloud Appliance
+
+A self-contained, single-machine deployment variant of Anbox Cloud.
+
+See [Variants](https://discourse.ubuntu.com/t/about-anbox-cloud/17802#variants).
+
+<a name="amc"></a>
+### Anbox Management Client (AMC)
+
+The command line interface that is used to manage the Anbox Management Service.
+
+<a name="ams"></a>
+### Anbox Management Service (AMS)
+
+The service that handles all aspects of the application and container life cycle in Anbox Cloud. AMS is responsible for managing containers, applications, addons, updates and more, ensuring high density, performance and fast container startup times.
```

Comment:
I would add a sentence explaining that AMS is using etcd as database and is connecting to LXD over its REST API